client/my-sites/github-deployments/controller.tsx

```diff
@@ -0,0 +1,34 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { PageViewTracker } from 'calypso/lib/analytics/page-view-tracker';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { GithubDeployments } from './main';
+import type { Callback } from '@automattic/calypso-router';
+
+export const githubDeployments: Callback = ( context, next ) => {
+	context.primary = (
+		<>
+			<PageViewTracker path="/github-deployments/:site" title="Github Deployments" delay={ 500 } />
+			<GithubDeployments />
```

Comment:
Maybe just me, but I think it'd be preferable to capitalise Github - ie. `<GitHubDeployments />`

I feel that's particularly important for strings shown on the front-end in the other file and on the sidebar.

Reply:
You are correct, It should be capitalised.